test/e2e/service_to_service_test.go

```diff
@@ -261,11 +262,17 @@ func TestSvcToSvcViaActivator(t *testing.T) {
 
 	clients := Setup(t)
 
+	mesh, _ := strconv.ParseBool(os.Getenv("MESH"))
```

Comment:
Can you add a flag via here?

https://github.com/knative/serving/blob/96c7bc172749474a8d0a5525e6eee74dab2967d1/test/e2e_flags.go#L33-L41

I want to avoid a sprawl of accessing env vars in tests vs. flags that make it more obvious what can be configured